stl/inc/vector

```diff
@@ -489,37 +494,38 @@ public:
 
 private:
 #ifdef _INSERT_VECTOR_ANNOTATION
-    static _CONSTEXPR20 void _Create_annotation(const vector& _Myvec_) noexcept {
+    _CONSTEXPR20 void _Create_annotation() const noexcept {
         // Annotates the shadow memory of the valid range
-        auto& _My_data = _Myvec_._Mypair._Myval2;
+        auto& _My_data = _Mypair._Myval2;
         _Apply_annotation(_My_data._Myfirst, _My_data._Myend, _My_data._Myend, _My_data._Mylast);
     }
 
-    static _CONSTEXPR20 void _Clear_annotation(const vector& _Myvec_) noexcept {
+    _CONSTEXPR20 void _Remove_annotation() const noexcept {
         // Removes the shadow memory annotation of the range
-        auto& _My_data = _Myvec_._Mypair._Myval2;
+        auto& _My_data = _Mypair._Myval2;
         _Apply_annotation(_My_data._Myfirst, _My_data._Myend, _My_data._Mylast, _My_data._Myend);
     }
 
-    static _CONSTEXPR20 void _Change_annotation(const vector& _Myvec_, const difference_type _Count) noexcept {
-        // Extends /shrinks the annotated range by _Count
-        auto& _My_data = _Myvec_._Mypair._Myval2;
+    _CONSTEXPR20 void _Modify_annotation(const difference_type _Count) const noexcept {
+        // Extends/shrinks the annotated range by _Count
+        auto& _My_data = _Mypair._Myval2;
         _Apply_annotation(_My_data._Myfirst, _My_data._Myend, _My_data._Mylast, _My_data._Mylast + _Count);
     }
 
-    static const void* _Get_aligned_first(const void* _First, const void* _End) noexcept {
-        const char* _CFirst    = reinterpret_cast<const char*>(_First);
-        const char* _CEnd      = reinterpret_cast<const char*>(_End);
+    _NODISCARD static const void* _Get_aligned_first(const void* _First, const void* _End) noexcept {
+        const auto _CFirst     = reinterpret_cast<const char* const>(_First);
+        const auto _CEnd       = reinterpret_cast<const char* const>(_End);

```

Comment:
The casts should be `reinterpret_cast<const char*>`, because top-level `const` has no effect in such casts.

Occurs below, and also affects `reinterpret_cast<const uintptr_t>`.